src/include/miopen/solver_base.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
```

Comment:
I suggest not to put this in a separate file for now, but to do it later, when the implementation of the new interface for solvers is completed.

Reply:
The file was redundant and has been removed.